doc/docs.md

```diff
@@ -2424,6 +2424,21 @@ p := &Point{10, 10}
 println(p.x)
 ```
 
+Struct fields can re-use reserved keywords:
+
+```v
+struct Employee {
+	type string
+	name string
+}
+
+employee := Employee{
+	type: 'FTE'
+	name: 'John Doe'
+}
+println(employee.type)
+```
+
```

Comment:
The description is very good, but I think that it should be moved at line 2410, before the `### Heap structs` heading, since it applies to all structs, not just to the heap tagged ones.